nationalsCodev1.ino

```diff
@@ -1,85 +1,95 @@
+/*
+White Station High School Science Olympiad Electric Vehicle Code
+  Written by Justin Du & Nilai Vemula 
+*/
 
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-///Version 1 of Science Olympiad Electric Vehicle Code//////////////////////////////////////////////////////////////////////////////////////////////////////
-///White Station High School////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-///Code Written by Nilai Vemula/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-//VERY IMPORTANT SETUP INFO FIRST///////////////////////////////////////////////////////////////////////////////
-  //Values to enter//
+/* Stuff to change at competition */
+// Distance where car should switch from fast to slow
+const long CHANGE_SPEED_DISTANCE = 8.5;
+// Final distance
+const long FINAL_DISTANCE = 9.5;
 
-const long change_speed_distance = 8.5;    ////Distance at where the car should switch from fast to slow (8.5m) ////
-const long final_distance = 9.5;     ////INSERT FINAL DISTANCE ////
-/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-
-
-//Wiring Information///////////////////////////////////////////////////////////////////////////////////////////////
-  //Encoder//
+/* Wiring Information */
 
+  /* Encoder */
     // Red - 5V
     // Black - GND
-    //Signal A in D2
-    const int encoder_a = 2; 
-    //Signal B in D3
-    const int encoder_b = 3; 
+    // Signal A in D2
+    const int ENCODER_A = 2; 
+    // Signal B in D3
+    const int ENCODER_B = 3; 
```

Comment:
I need to confirm with the vehicle. It is clockwise/counter-clockwise, not left/right.